### DISCUSSION TITLE
Override w-full with w-fit

### DISCUSSION THREAD

**Discussion by AkshatGiri** [dcastil/tailwind-merge#110] [Help] [answered]
I'm trying to override the w-full property with w-fit but currently the function is return both values.

Input = `twMerge('w-full', 'w-fit')`

Current output = `w-full w-fit`

Expected result = `w-fit`

Curious to see if this is a bug or expected behaviour. 

`"tailwind-merge": "^1.3.0"`

**Comment by dcastil** [comment, marked as answer]
Oh yes, this is a bug. I'll fix this soon.

**Reply by dcastil** [reply]
Tracking this in https://github.com/dcastil/tailwind-merge/issues/111.

**Reply by dcastil** [reply]
@AkshatGiri your issue should be fixed in [`v1.4.0`](https://github.com/dcastil/tailwind-merge/releases/tag/v1.4.0).

**Reply by AkshatGiri** [reply]
Thank you for the fix!
